voting-contentApp/apps/voting-app.yaml

```diff
@@ -0,0 +1,31 @@
+!content-type:poll
+name: voting-app
+label: Voting
+icon: icon-segments-app
+
+subApps:
+  browser:
+    workbench:
+      contentViews: !override
+        - name: tree !override
+          multiSelect: true
+          $type: treeView
+          dropConstraint:
+            $type: jcrDropConstraint
+            primaryNodeType: mgnl:composition
+          columns: &columns
+            - name: jcrName
+              label: Poll
+              $type: jcrTitleColumn
+            - name: startdate
+              $type: dateColumn
+            - name: expdate
+              $type: dateColumn
+              nodeTypeToIcon:
+                mgnl:folder: icon-folder
+            - name: pollstatus
+              filterComponent:
+                $type: comboBoxField
+            - name: mgnl:lastModified
+              label: Modification Date
```

Comment:
Should apply i18n key for label.

Reply:
I'm not sure if i18n is supported for labels, since there was issue regarding to this. Keep it as is if it's not working

Reply:
As Mr. Khanh's opinion, I cannot apply i18n by using voting-app.browser.views.mgnl:lastModified.label=Modification Date and I have not found the solution yet so I use the label property to make it work.

Reply:
Actually we can. However, I accept to use as default at the moment. As I remember that there're fixed label for predefined properties such as name, modification date...e.t.c... we only need to override them.